app/controllers/items_controller.rb

```diff
@@ -19,6 +19,16 @@ def create
     end
   end
 
+  def show
+    @item = Item.find(params[:id])
+    @user = User.find(@item.user_id)
+  end
+
+  def edit
+    @item = Item.find(params[:id])
+  end
```

Comment:
```
def edit
  @item = Item.find(params[:id])
end
```
こちらは商品編集の実装範囲になるかと存じます。
現状としてはコメントアウトもしくは削除しておきましょう！
理由としては商品編集の実装時に差分として確認できなくなる可能性があるためです。

Reply:
承知しました。
コメントアウトするように致します。

```suggestion
  #def edit
    #@item = Item.find(params[:id])
  #end
```